accounts/migrations/0041_add_existing_conditions.py

```diff
@@ -0,0 +1,256 @@
+    "na": None,
+    "<24": 0,
+    "24": 1,
+    "25": 2,
+    "26": 3,
+    "27": 4,
+    "28": 5,
+    "29": 6,
+    "30": 7,
+    "31": 8,
+    "32": 9,
+    "33": 10,
+    "34": 11,
+    "35": 12,
+    "36": 13,
+    "37": 14,
+    "38": 15,
+    "39": 16,
+    "40>": 17,
+}
+
+
+REVERSE_GESTATIONAL_AGE_MAPPING = {
+    v: k for k, v in GESTATIONAL_AGE_CHAR_TO_INT_MAPPING.items()
+}
+
+
+def _forwards_migrate_ga_birth_char_field_to_int(apps, schema_editor):
+
+    ChildModel = apps.get_model("accounts", "Child")
+
+    for child in ChildModel.objects.all():
+        child.gestational_age_at_birth = GESTATIONAL_AGE_CHAR_TO_INT_MAPPING[
+            child.age_at_birth
+        ]
+        child.save()
+
+
+def _backwards_reverse_int_field_to_char(apps, schema_editor):
+    ChildModel = apps.get_model("accounts", "Child")
+
+    for child in ChildModel.objects.all():
+        child.age_at_birth = REVERSE_GESTATIONAL_AGE_MAPPING[
+            child.gestational_age_at_birth
+        ]
+        child.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("accounts", "0040_add_message_model")]
+
+    operations = [
+        migrations.AddField(
+            model_name="child",
+            name="gestational_age_at_birth",
+            field=models.PositiveSmallIntegerField(
+                choices=[
+                    (None, "Not sure or prefer not to answer"),
+                    (0, "Under 24 weeks"),
+                    (1, "24 weeks"),
+                    (2, "25 weeks"),
+                    (3, "26 weeks"),
+                    (4, "27 weeks"),
+                    (5, "28 weeks"),
+                    (6, "29 weeks"),
+                    (7, "30 weeks"),
+                    (8, "31 weeks"),
+                    (9, "32 weeks"),
+                    (10, "33 weeks"),
+                    (11, "34 weeks"),
+                    (12, "35 weeks"),
+                    (13, "36 weeks"),
+                    (14, "37 weeks"),
+                    (15, "38 weeks"),
+                    (16, "39 weeks"),
+                    (17, "40 or more weeks"),
+                ],
+                default=None,
+                null=True,
+            ),
+        ),
+        migrations.RunPython(
+            _forwards_migrate_ga_birth_char_field_to_int,
+            reverse_code=_backwards_reverse_int_field_to_char,
+        ),
+        migrations.AlterField(  # manually added to enable migration reversal, see https://stackoverflow.com/a/28879030
+            model_name="child",
+            name="age_at_birth",
+            field=models.CharField(
+                max_length=25, choices=AGE_AT_BIRTH_CHOICES, default="na"
+            ),
+            preserve_default=False,
+        ),
+        migrations.RemoveField(model_name="child", name="age_at_birth"),
+        migrations.AddField(
+            model_name="child",
+            name="existing_conditions",
+            field=bitfield.models.BitField(
```

Comment:
Align this with the shorter list?

Reply:
This is an intermediate migration - see the later files for the lambda deferred evaluation to trick Django migrations into being smarter about Bitfield changes